### DISCUSSION TITLE
Octernship Number of Contribution To be Eligible

### DISCUSSION THREAD

**Discussion by Shivam7-1** [community/community#63693] [GitHub Education]
### Hi everyone,

Hello ,

So far, I have made 94 Contribution in this Year However, despite these contributions, I am still receiving the message, "Oh oh, looks like you need to contribute a bit more to the GitHub community before you’re eligible for an Octernship".

I would like to know the minimum number of contributions we need to make per month/year to become eligible for this program.

**Comment by Sooryak12** [comment]
did you figure it out ?

**Comment by Shr11** [comment]
They have their own criteria set I guess..
